Content and layout for Giving Tuesday 2022

coderbyheart opened the pull request:
This adds the necessary Forestry content and layout for the new donation page.

Preview: https://deploy-preview-699--da-landing-2.netlify.app/donate/

Project: https://github.com/orgs/distributeaid/projects/6

## Remaining tasks

- [x] remove allocated funds on all projects 
- [x] add padding on mobile 

(can be addressed in a separate PR)

- [x] improve display of images in Gallery

## Design

### Donate page

<img src="https://user-images.githubusercontent.com/188915/197054890-d8a797ef-6351-45f3-b8fb-839f7a536a88.jpg" alt="" width="250" />

### Breakout page

<img src="https://user-images.githubusercontent.com/188915/197054925-55eced3d-be00-432b-8a64-d8b8b492f41e.jpg" alt="" width="250" />

- [x] #784

### Donate options

![Donate options](https://user-images.githubusercontent.com/188915/196795665-259437f1-d648-411a-9d9d-df28d2a5ad1e.png)

## Forestry

Branch for forestry: `giving-tuesday-2022`

Fundraisers are managed in Forestry:

![image](https://user-images.githubusercontent.com/188915/193069537-75b18f20-11b9-4c7f-9278-0737434e37a2.png)

ghost commented:
<details>
<summary><b>👇 <a href="https://app.codesee.io/r/reviews?pr=699&src=https%3A%2F%2Fgithub.com%2Fdistributeaid%2Fdistributeaid.org">Click on the image</a> for a new way to code review</b>
</summary>

  - Make big changes easier — review code in small groups of related files
  - Know where to start — see the whole change at a glance
  - Take a code tour — explore the change with an interactive tour
  - Make comments and review — all fully sync’ed with github

    [Try it now!](https://app.codesee.io/r/reviews?pr=699&src=https%3A%2F%2Fgithub.com%2Fdistributeaid%2Fdistributeaid.org)
</details>

[![Review these changes using an interactive CodeSee Map](https://s3.us-east-2.amazonaws.com/maps.codesee.io/images/github/distributeaid/distributeaid.org/699/3eaf4a7b/54fe0a37b9e4ced9ee2df859ae4d13566cf5b377.svg)](https://app.codesee.io/r/reviews?pr=699&src=https%3A%2F%2Fgithub.com%2Fdistributeaid%2Fdistributeaid.org)

#### Legend
![CodeSee Map Legend](https://app.codesee.io/Legend_v2.png)

jtfairbank commented:
Whoops!  @coderbyheart heads up that I've pushed here.  Was working on styling the new Fundraisers pages according to the designs, but meant to push to my own branch and open up a work-in-progress PR to merge those changes here.

I'd say fI'm about halfway there with the basic styles for the fundraiser pages tho.  Got a good page skeleton & theming going.

coderbyheart commented:
@jtfairbank I've pushed your changes to a separate branch: https://github.com/distributeaid/distributeaid.org/tree/giving-tuesday-2022-taylor

coderbyheart commented:
![Screenshot_2022-11-10-19-12-41-61_40deb401b9ffe8e1df2f1cc5ba480b12](https://user-images.githubusercontent.com/188915/201174529-f12728fe-f6f0-4193-b048-f794575212c6.jpg)

![Screenshot_2022-11-10-19-12-58-09_40deb401b9ffe8e1df2f1cc5ba480b12](https://user-images.githubusercontent.com/188915/201174555-fcbb807a-0293-4d62-b41a-8d78e1a03f87.jpg)

coderbyheart commented:
![Screenshot_2022-11-10-19-12-30-45_40deb401b9ffe8e1df2f1cc5ba480b12](https://user-images.githubusercontent.com/188915/201174713-f47aa745-9139-4eb1-a4d5-e49dbdc76d42.jpg)

jtfairbank commented on static/admin/index.html:
```diff
@@ -26,7 +26,7 @@
 <script id="admin-config-script" type="text/javascript">
   var env = {
-      siteId: "0pvklfck1gkdpg",
+      siteId: "g0tpjekrmqgtkw",
```
Note: Might not want to merge the updated Forestry site ID, since changes will be available in the "Saga" site on Forestry after merging the PR.

jtfairbank commented on src/templates/FundraiserPage.tsx:
```diff
@@ -0,0 +1,14 @@
+import { Fundraiser } from '@components/fundraiser/Fundraiser'
```
NOTE: Use Gatsby's filesystem route API to generate pages, instead of templates.

See [#623](https://github.com/distributeaid/distributeaid.org/issues/623) for context.

jtfairbank commented on src/stylesheets/donate.css:
```diff
@@ -0,0 +1,183 @@
+main.donate {
```
NOTE: convert to tailwind styles

jtfairbank commented:
@coderbyheart review in progress!  Will take me a few days so I'm leaving notes for myself, don't worry about anything for now I'll provide feedback in a comment down the road using Conventional Comments.

coderbyheart committed:
feat: add content for Giving Tuesday

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/blocks/fundraisers/us-disaster-relief.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/blocks/fundraisers/us-disaster-relief.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/blocks/fundraisers/ukraine-response.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/blocks/fundraisers/ukraine-response.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/blocks/fundraisers/tech-for-good.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/blocks/fundraisers/sustainable-supply-chains.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/blocks/fundraisers/humanitarian-logistics.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/blocks/fundraisers/humanitarian-logistics.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/blocks/fundraisers/european-refugee-relief.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/blocks/fundraisers/us-disaster-relief.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/blocks/fundraisers/us-disaster-relief.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/blocks/fundraisers/ukraine-response.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/blocks/fundraisers/tech-for-good.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/blocks/fundraisers/sustainable-supply-chains.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/blocks/fundraisers/humanitarian-logistics.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/blocks/fundraisers/european-refugee-relief.md

jtfairbank committed:
Contributions from Taylor's Review of giving-tuesday-2022 (#840)

coderbyheart committed:
fix: remove donate.css

coderbyheart committed:
fix: revert site ID change
